build/docker/structure-test.yaml

```diff
@@ -14,8 +14,7 @@ commandTests:
   - name: "journalctl is working in the image"
     command: "journalctl"
     expectedOutput: ["-- No entries --\n"]
-    excludedError: ["No journal files were found.\n"]
```

Comment:
Removing the test. The `stderr` of the `journalctl` contained the `No journal files were found.` message even before this PR.
For some reason, the test did not detect it before. Now, with the new `journalctl` binary, the message is rendered differently in the terminal and the test started detecting it.